src/main/java/com/google/devtools/build/lib/skyframe/BUILD

```diff
@@ -2801,3 +2802,9 @@ java_library(
         "//third_party:jsr305",
     ],
 )
+
+java_library(
+    name = "already_reported_exception",
+    srcs = ["AlreadyReportedException.java"],
+    deps = [],
```

Comment:
nit: can we just leave out an empty deps attribute?

Reply:
Done.